projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/scan-threats.tsx

```diff
@@ -0,0 +1,213 @@
+												noArrow={ false }
+												offset={ 10 }
+												focusOnMount={ 'container' }
+												onClose={ hideTooltip }
+											>
+												<>
+													<h3 className="value-section__tooltip-heading">
+														{ scanThreatsTooltip.title }
+													</h3>
+													<p className="value-section__tooltip-content">
+														{ scanThreatsTooltip.text }
+													</p>
+												</>
+											</Popover>
+										) }
+									</div>
+								</div>
+							</div>
+						</>
+					);
+				}
+				return (
+					<>
+						<div className="value-section__heading">{ __( 'Threats', 'jetpack-my-jetpack' ) }</div>
+						<div className="value-section__data">
+							<div className="scan-threats__threat-count">{ numThreats }</div>
+						</div>
+					</>
+				);
+			}
+			return (
+				<>
+					<div className="value-section__heading">{ __( 'Scan', 'jetpack-my-jetpack' ) }</div>
+					<div className="value-section__data">
+						<div>
+							<img
+								className="value-section__status-icon"
+								src={ ShieldSuccess }
+								alt={ __( 'Shield icon - Scan Status: Secure', 'jetpack-my-jetpack' ) }
+							/>
+						</div>
+						<div className="value-section__status-text">
+							{ __( 'Secure', 'jetpack-my-jetpack' ) }
+						</div>
+					</div>
+				</>
+			);
+		}
+		return numThreats ? (
+			<>
+				<div className="value-section__heading">{ __( 'Threats', 'jetpack-my-jetpack' ) }</div>
+				<div className="value-section__data">
+					<div className="scan-threats__threat-count">{ numThreats }</div>
+				</div>
+			</>
+		) : (
+			<>
+				<div className="value-section__heading">{ __( 'Scan', 'jetpack-my-jetpack' ) }</div>
+				<div className="value-section__data">
+					<div>
+						<img
+							className="value-section__status-icon"
+							src={ ShieldPartial }
+							alt={ __( 'Shield icon - Scan Status: Partial', 'jetpack-my-jetpack' ) }
+						/>
+					</div>
+					<div className="value-section__status-text">
+						{ __( 'Partial', 'jetpack-my-jetpack' ) }
+					</div>
+					<InfoTooltip
+						tracksEventName={ 'protect_card_tooltip_open' }
+						tracksEventProps={ {
+							location: 'scan',
+							status: 'partial',
+							hasPaidPlan: false,
+							threats: 0,
+						} }
+					>
+						<>
+							<h3 className="value-section__tooltip-heading">{ scanThreatsTooltip.title }</h3>
+							<p className="value-section__tooltip-content">{ scanThreatsTooltip.text }</p>
+						</>
+					</InfoTooltip>
+				</div>
+			</>
+		);
+	}
+
+	return (
+		<>
+			<div className="value-section__heading">{ __( 'Scan', 'jetpack-my-jetpack' ) }</div>
+			<div className="value-section__data">
+				<div>
+					<img
+						className="value-section__status-icon"
+						src={ ShieldOff }
+						alt={ __( 'Shield icon - Scan Status: Off', 'jetpack-my-jetpack' ) }
+					/>
+				</div>
+				<div className="value-section__status-text">{ __( 'Off', 'jetpack-my-jetpack' ) }</div>
+			</div>
+		</>
+	);
+};
```

Comment:
There are many levels in this code. For readability, we could think of splitting it into e.g. small components to reduce complexity. Do you think it would be possible here?

Reply:
Yes, I'll split into smaller components. 👍